### PR TITLE
chore(react-scheduler): add the AppointmentTooltip plugin reference

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Describes properties passed to a component that renders the appointment tooltip command button.
+
+Field | Type | Description
+------|------|------------
+id | `open` &#124; `delete` &#124; `close` | The command identifier.
+onExecute | () => void | An event initiating the command execution.
+
+### AppointmentMeta
+
+An object representing a appointment tooltip meta data.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: 'a appointment' => 'an appointment'

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+children | ReactNode | A React node used to render a layout content.
+
+### AppointmentTooltip.HeaderProps
+
+Describes properties passed to a component that renders a tooltip header.
+
+Field | Type | Description
+------|------|------------
+appointmentData | object | The appointment's meta data to display.
+children | ReactNode | A React node used to render a tooltip header.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: It isn't a required prop.

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+onHide | () => void | An event that hides the tooltip.
+children | ReactNode | A React node used to render a layout content.
+
+### AppointmentTooltip.HeaderProps
+
+Describes properties passed to a component that renders a tooltip header.
+
+Field | Type | Description
+------|------|------------
+appointmentData | object | The appointment's meta data to display.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: It isn't a required prop.

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+children | ReactNode | A React node used to render a tooltip header.
+
+### AppointmentTooltip.ContentProps
+
+Describes properties passed to a component that renders a tooltip content.
+
+Field | Type | Description
+------|------|------------
+appointmentData | object | The appointment's meta data to display.
+children | ReactNode | A React node used to render a tooltip content.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: The same about these props.

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+### AppointmentTooltip.CommandButtonProps
+
+Describes properties passed to a component that renders a tooltip's command button.
+
+Field | Type | Description
+------|------|------------
+id | `open` &#124; `delete` &#124; `close` | The command identifier.
+onExecute | () => void | An event that executes the command.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: The same about these props.

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
+# AppointmentTooltip Plugin Reference
+
+The `AppointmentTooltip` plugin allows you to display a brief information about an appointment.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a brief information?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { AppointmentTooltip } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: Same change here.

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+## Interfaces
+
+### AppointmentTooltip.LayoutProps
+
+Describes properties passed to a component that renders a tooltip layout.
+
+Field | Type | Description
+------|------|------------
+headerComponent | ComponentType&lt;[AppointmentTooltip.HeaderProps](#appointmenttooltipheaderprops)&gt; | A component that renders a tooltip header.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a tooltip header?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+contentComponent | ComponentType&lt;[AppointmentTooltip.ContentProps](#appointmenttooltipcontentprops)&gt; | A component that renders a  tooltip content.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a  tooltip content?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+Describes properties passed to a component that renders a tooltip header.
+
+Field | Type | Description
+------|------|------------
+appointmentData | object | The appointment's meta data to display.
+children | ReactNode | A React node used to render a tooltip header.
+
+### AppointmentTooltip.ContentProps
+
+Describes properties passed to a component that renders a tooltip content.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a tooltip content?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+children | ReactNode | A React node used to render a layout content.
+
+### AppointmentTooltip.HeaderProps
+
+Describes properties passed to a component that renders a tooltip header.
+
+Field | Type | Description
+------|------|------------
+appointmentData | object | The appointment's meta data to display.
+children | ReactNode | A React node used to render a tooltip header.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a tooltip header?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+onExecute | () => void | An event that executes the command.
+
+### AppointmentMeta
+
+An appointment's meta data object.
+
+Field | Type | Description
+------|------|------------
+target | ReactInstance | A React component instance or a DOM element that is used to position the tooltip.
+data | object | The appointment's meta data to display.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: The appointment's meta data to display? The appointment's displayed metadata?

File: packages/dx-react-scheduler/docs/reference/appointment-tooltip.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+AppointmentTooltip.Layout | ComponentType&lt;[AppointmentTooltip.LayoutProps](#appointmenttooltiplayoutprops)&gt; | A component that renders a tooltip layout.
+AppointmentTooltip.Header | ComponentType&lt;[AppointmentTooltip.HeaderProps](#appointmenttooltipheaderprops)&gt; | A component that renders a tooltip header.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/appointment-tooltip.md]: a tooltip header?

**Pull request merged**